main.go

```diff
@@ -38,6 +38,7 @@ import (
 
 var configScanDir = []string{
 	"/oem",
+	"/system/oem",
```

Comment:
I wonder what the relation with these paths is: https://github.com/kairos-io/immucore/blob/a2874ca3ee5385a0e81a9e9f8c95633939877656/internal/constants/constants.go#L11

If they are supposed to be the same paths, maybe it's worth moving them to the kairos-sdk and use them from there to make sure they never diverge.

Reply:
good call, wasn't aware of these